cwclient/cwclient.go

```diff
@@ -95,6 +95,44 @@ func (cwClient *CosmWasmClient) QueryIsEnabled() (bool, error) {
 	return isEnabled, nil
 }
 
+func (cwClient *CosmWasmClient) QueryIsBlockForked(blockHeight uint64) (bool, error) {
+	queryData, err := createIsForkedBlockQueryData(blockHeight)
+	if err != nil {
+		return false, err
+	}
+
+	resp, err := cwClient.querySmartContractState(queryData)
+	if err != nil {
+		return false, err
+	}
+
+	var isForked bool
+	if err := json.Unmarshal(resp.Data, &isForked); err != nil {
+		return false, err
+	}
+
+	return isForked, nil
+}
+
+func (cwClient *CosmWasmClient) QueryForkedBlocksInRange(startHeight, endHeight uint64) ([]blockRange, error) {
```

Comment:
who will call this function? i don't see it's used

Reply:
yes it's not called by FG in the latest implementation, but i kept it here for now. We can delete it once we have something working e2e

Reply:
got it. let's add a comment to mention it's for tmp testing only?